master/docs/manual/credentialmanagement.rst

```diff
@@ -0,0 +1,100 @@
+Credential Management
+=====================
+
+Definition
+----------
+
+Buildbot needs to send and/or execute commands requiring credentials.
+
+The goal is to store credentials used by Buildbot commands.
+
+Credential could not be stored in ``buildbot.config`` or ``master.config`` because the configuration is available in a source control management.
+
+Credential are stored in the Buildbot database.
+
+Credentials once stored are obfuscated.
```

Comment:
Credentials are never visible to the normal user via logs, and thus are transmitted to the workers using the ```:class:`Obfuscated` ```